source/features/preview-hidden-comments.tsx

```diff
@@ -29,9 +29,9 @@ function init(): void {
 		}
 
 		// Hidden review comments that have been preloaded have their header text wrapped in an extra <div>
-		const headerTextWrapper = header.tagName === 'H3' ? select(':scope > .d-inline-block:last-child', header) : undefined;
+		const headerTextWrapper = header.tagName === 'H3' ? select(':scope > .d-inline-block:last-child', header) : header.firstChild;
 		header.append(
-			<span className="Details-content--open">{headerTextWrapper ?? header.firstChild}</span>,
+			<span className="Details-content--open">{headerTextWrapper}</span>,
```

Comment:
Note: This isn't equivalent. The previous code worked even when the select was empty. 

Not sure if it's still fine

Reply:
Yes I remember doing this on purpose. Though this could make clearer, at the code of a little redundancy:
```ts
const headerTextWrapper = header.tagName === 'H3' ? select(':scope > .d-inline-block:last-child', header) ?? header.firstChild : header.firstChild
```